easybuild/easyconfigs/f/fastqc/fastqc-0.11.4-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'EB_Java'
```

Comment:
I'll make the changes to this fastqc and then change the older fastqc 
easyconfigs, on which I based this one.

BTW: there are older easyconfigs with different capitalisation (FastQC), 
should I make this consistent across them all?

On 06/01/2016 22:13, Kenneth Hoste wrote:
>
> In easybuild/easyconfigs/f/fastqc/fastqc-0.11.4-Java-1.8.0_66.eb 
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/2267#discussion_r48949343>:
>
> > @@ -0,0 +1,28 @@
> > +easyblock = 'EB_Java'
>
> The only thing that |EB_Java| does on top of |PackedBinary| is to 
> update |$JAVA_HOME|.
>
> Using the |fastqc| install prefix to define |$JAVA_HOME| is probably 
> very wrong?
>
> So, |PackedBinary| should be used here instead (and in other |fastqc| 
> easyconfigs)?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/2267/files#r48949343>.
>



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus